v20.1/admin-ui-storage-dashboard.md

```diff
@@ -1,35 +1,57 @@
 ---
 title: Storage Dashboard
-summary: The Storage dashboard lets you monitor the storage utilization for your cluster.
+summary: The Storage dashboard lets you monitor the storage utilization of your cluster.
 toc: true
 ---
 
-The **Storage** dashboard in the CockroachDB Admin UI lets you monitor the storage utilization for your cluster. To view this dashboard, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui), click **Metrics** on the left-hand navigation bar, and then select **Dashboard** > **Storage**.
+The **Storage** dashboard lets you monitor the storage utilization of your cluster. 
+
+To view this dashboard, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui), click **Metrics** in the left-hand navigation, and select **Dashboard** > **Storage**.
 
 {% include {{ page.version.version }}/admin-ui/admin-ui-metrics-navigation.md %}
 
 The **Storage** dashboard displays the following time series graphs:
 
 ## Capacity
 
+You can monitor the **Capacity** graph to determine when additional storage is needed (e.g., by scaling your cluster). 
```

Comment:
Same as above, can "scaling your cluster" link to instructions for adding nodes?